[TT-9022] Restructured and clarified caching documentation (take 2) (#2687)
TykTechnologies/tyk-docs: tyk-docs/content/basic-config-and-security/reduce-latency/caching/optimise-cache.md

---
title: "Optimising the Cache Storage"
date: 2023-06-08
tags: ["Optimise Cache Storage", "Cache", "Caching", "Redis", "Advanced"]
description: ""
menu:
  main:
    parent: "Caching"
weight: 6
---

Tyk creates the API cache in Redis, as it gives high performance and low latency. By default, the cache will use the same database that is used to store the API keys, minimising the deployment footprint.

For [multi-data centre]({{< ref "tyk-multi-data-centre/mdcb-components#redis">}}) deployments, the Data Planes have a locally deployed Redis. This enables them to have a localised cache close to the traffic-serving Gateways.

The [cache key]({{< ref "basic-config-and-security/reduce-latency/caching#cache-key">}}) is used as the Redis key, for quick lookups.

> For high-traffic systems that make heavy use of caching as well as rate limiting, it can make sense to use a separate Redis database for cache storage, at the expense of increased deployment footprint.

Author: ```suggestion
For high-traffic systems that make heavy use of caching, it can make sense to use separate Redis databases for cache storage and for API keys, at the expense of increased deployment footprint.
```
